test/github-issues/1780/issue-1780.ts

```diff
@@ -0,0 +1,176 @@
+                    .values(data)
+                    .execute();
+
+                await connection.manager
+                    .createQueryBuilder()
+                    .insert()
+                    .onConflict("DO NOTHING") 
+                    .into(MultipleUniqueIndex)
+                    .values(data)
+                    .execute();
+                
+                let loadedSimpleUniqueIndexEntity = await connection.manager.find(SimpleUniqueIndex);
+                expect(loadedSimpleUniqueIndexEntity).not.to.be.empty;
+                loadedSimpleUniqueIndexEntity.length.should.be.equal(2);
+
+                let loadedMultipleUniqueIndexEntity = await connection.manager.find(MultipleUniqueIndex);
+                expect(loadedMultipleUniqueIndexEntity).not.to.be.empty;
+                loadedMultipleUniqueIndexEntity.length.should.be.equal(2);
+            }
+        }
+        catch (err) {
+
+          throw new Error(err);
+        }
+    })));
+
+    it("should save two rows without duplicate error in Oracle Database", () => Promise.all(connections.map(async connection => {
+
+        try {
+            if (connection.driver instanceof OracleDriver) {
+
+                await connection.manager
+                    .createQueryBuilder()
+                    .insert()
+                    .onConflict("/*+ ignore_row_on_dupkey_index(simple_unique_index, unique-idx) */")
+                    .into(SimpleUniqueIndex)
+                    .values(data)
+                    .execute();
+
+                await connection.manager
+                    .createQueryBuilder()
+                    .insert()
+                    .onConflict("/*+ ignore_row_on_dupkey_index(multiple_unique_index, unique-idx) */")
+                    .into(MultipleUniqueIndex)
+                    .values(data)
+                    .execute();
+                
+                let loadedSimpleUniqueIndexEntity = await connection.manager.find(SimpleUniqueIndex);
+                expect(loadedSimpleUniqueIndexEntity).not.to.be.empty;
+                loadedSimpleUniqueIndexEntity.length.should.be.equal(2);
+
+                let loadedMultipleUniqueIndexEntity = await connection.manager.find(MultipleUniqueIndex);
+                expect(loadedMultipleUniqueIndexEntity).not.to.be.empty;
+                loadedMultipleUniqueIndexEntity.length.should.be.equal(2);
+                             
+            }
+        }
+        catch (err) {
+
+          throw new Error(err);
+        }
+    })));
+
+    it("should save two rows without duplicate error in SQLite/Sqljs", () => Promise.all(connections.map(async connection => {
+
+        try {
+            if ([SqljsDriver, SqliteDriver].find(currentDriver => connection.driver instanceof currentDriver)) {
+
+                await connection.manager
+                    .createQueryBuilder()
+                    .insert()
+                    .onConflict("OR IGNORE") 
```

Comment:
Maybe its better to add `orIgnore` method into query builder? Because this string looks weird. If we do so, then we can use this method in mysql as well.

Reply:
Ok, I'll update PR with orIgnore and orUpdate methods.